bsp4j/src/main/java/ch/epfl/scala/bsp4j/ScalaExtension.xtend

```diff
@@ -12,13 +12,15 @@ class ScalaBuildTarget {
   @NonNull String scalaBinaryVersion
   @NonNull ScalaPlatform platform
   @NonNull List<String> jars
+  @NonNull JvmBuildTarget jvmBuildTarget
```

Comment:
I like this design over `ScalaBuildTarget extends JvmBuildTarget`. Composition is better than inheritance in this case 👍